moodle/Tests/Sniffs/Commenting/fixtures/ValidTags/general.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * A fixture to verify various phpdoc tags in a general location.
+ *
+ * @package   local_moodlecheck
+ * @copyright 2018 onwards Eloy Lafuente (stronk7) {@link http://stronk7.com}
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class some_class extends \advanced_testcase {
+    /**
+     * Some valid tags, to verify they are ok.
+     *
+     * @license
+     * @throws
+     * @deprecated
+     * @author
+     * @todo
+     */
+    public function all_valid_tags() {
+        echo "yay!";
+    }
+
+    /**
+     * Some invalid tags, to verify they are detected.
+     *
+     * @codingStandardsIgnoreLine
```

Comment:
This is invalid, isn't it? Not sure to understand why line # 27 is not in the expectations, uhm... what am I missing?

More yet, maybe we should consider (not as part of this PR!), to find a way to migrate from the old annotations to the new ones, as far as the old ones are going to stop working at some point in the future ([link](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file)).

Reply:
It wasn't detected by moodlecheck so I initially didn't cover it. I tried to have it rename the line now, but it can't because the line is self-ignored!

Reply:
Aha, that explains why the original test in local_moodlecheck was detecting it as wrong (because it's not running phpcs, but moodlecheck, hah. Didn't realised!).